app/grants/views.py

```diff
@@ -604,3 +608,20 @@ def leaderboard(request):
             params['items'].append(item)
             counter += 1
     return TemplateResponse(request, 'grants/leaderboard.html', params)
+
+
+def record_grant_activity_helper(activity_type, subscription, profile):
```

Comment:
this should also be called in grant.models.py:subscription.successful_contribution for the `new_grant` activity type.

Reply:
@owocki, the `new_grant` activity type is meant for when a user creates a grant, not when they subscribe to it. So it has nothing to do with the feature I implemented.